src/main/resources/assets/tfc/lang/en_us.lang

```diff
@@ -87,6 +87,7 @@ tfc.tooltip.oredictionaryentry=§fOre Dictionary name:§7 %s
 tfc.tooltip.oredictionaryentries=§fOre Dictionary names:
 tfc.tooltip.metal=§fMetal:§7 %s
 tfc.tooltip.units=%d units
+tfc.tooltip.melttemp=%d°
```

Comment:
This opens up debate of which units we use, which for me seems to be Celsius. If so:
```suggestion
tfc.tooltip.melttemp=%d °C
```

Reply:
Shouldn't be a space between the value and the symbol. Whether to include the letter for the scale is entirely up to you

Reply:
Added without the space but with the C symbol